src/Guesser/AbstractTypeGuesser.php

```diff
@@ -28,10 +28,12 @@ abstract class AbstractTypeGuesser implements TypeGuesserInterface
     protected function getParentMetadataForProperty($baseClass, $propertyFullName, ModelManager $modelManager)
     {
         try {
-            return $modelManager->getParentMetadataForProperty($baseClass, $propertyFullName);
+            $metadata = $modelManager->getParentMetadataForProperty($baseClass, $propertyFullName);
```

Comment:
You can achieve the same result with fewer instructions and variables:
```suggestion
            return $modelManager->getParentMetadataForProperty($baseClass, $propertyFullName);
```

Reply:
It was like this before, but @phansys didn't like the way it was, in case someone add more catch and forget to return a value.

Reply:
This is [the thread](https://github.com/sonata-project/SonataDoctrineMongoDBAdminBundle/pull/359#discussion_r406655550) for those comments.